src/app/component/SliderWidget/SliderWidget.container.js

```diff
@@ -71,6 +72,12 @@ export class SliderWidgetContainer extends DataContainer {
     }
 
     render() {
+        const { isLoaded } = this.state;
+
+        if (!isLoaded) {
+            return null;
```

Comment:
Render `null` might cause jumps! How is that related to compare? :D

Reply:
This is from the merged commit 87ac9c9